docs/index.md

```diff
@@ -0,0 +1,36 @@
+Escoria official documentation
+==============================
+
+This page lists all existing documentation pages for the use of Escoria framework for Godot Engine.
+
+Installation
+------------
+
+[Git submodules](git_submodule.md)
+
+Reference
+---------
+
+[Background](background.md)
```

Comment:
There should be empty lines between these links so that they are not inlined.

Reply:
The same goes for the Scripting and Tools sections.

Reply:
Fixed